### PR TITLE
Migrate to NV-GHA Runners 

### PR DESCRIPTION
**Pull request by ArangoGutierrez**

### CODE REVIEW

File: deployments/gpu-operator/charts/node-feature-discovery-chart-0.16.2.tgz
Comment (tariq1890) [inline deployments/gpu-operator/charts/node-feature-discovery-chart-0.16.2.tgz:1]: Are we supposed to be adding this?

File: tests/kubeconfig.yml
Comment (tariq1890) [inline tests/kubeconfig.yml:1]: Why are we adding a kubeconfig file to the PR?

Ideally, we should only have a program accept the `KUBECONFIG` envar and supply the kubeconfig internally such that it's opaque to GH users

Comment (ArangoGutierrez) [reply tests/kubeconfig.yml:1]: my bad, dropped

File: deployments/gpu-operator/charts/node-feature-discovery-chart-0.16.2.tgz
Comment (ArangoGutierrez) [reply deployments/gpu-operator/charts/node-feature-discovery-chart-0.16.2.tgz:1]: my bad, dropped

**Comment by ArangoGutierrez** [comment]
PING @tariq1890 / @cdesiniotis

**Comment by ArangoGutierrez** [comment]
Test now pass
@tariq1890  / @cdesiniotis

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -15,16 +15,9 @@
...
-  pull_request:
-    types:
-      - opened
-      - synchronize
-    branches:
-      - main
-      - release-*
   push:
     branches:
-      - main
```
Comment (tariq1890) [inline .github/workflows/ci.yaml:27]: Why remove main? We also want to trigger e2e tests on pushes to main branch

Comment (ArangoGutierrez) [reply .github/workflows/ci.yaml:27]: k

Comment (ArangoGutierrez) [reply .github/workflows/ci.yaml:27]: Done

**Review by tariq1890 — APPROVED** [review]

**Pull request merged**